tsconfig.json

```
@@ -1,17 +1,12 @@
 {
   "compilerOptions": {
-    "declaration": true,
     "esModuleInterop": true,
-    "inlineSources": true,
     "lib": ["ES2020"],
     "module": "CommonJS",
-    "moduleResolution": "Node",
-    "outDir": "dist",
-    "rootDir": "src",
-    "sourceMap": true,
+    "moduleResolution": "node",
+    "noEmit": true,
     "strict": true,
-    "target": "ES2017"
+    "target": "es2017"
   },
-  "exclude": ["./src/**/*.test.ts"],
-  "include": ["./src/**/*.ts"]
+  "exclude": ["./dist/**"]
```

In the typescript docs, usually to exclude or include an entire directory recursively, this pattern is used:

```suggestion
  "exclude": ["./dist/**/*"]
```

Though I am uncertain whether these are evaluated differently or not :thinking:

---

I am uncertain as well! But I will make this change just to be safe.